src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -0,0 +1,26 @@
+xml.rss version: '2.0' do
+  xml.channel do
+    xml.title "#{@user.realname} (#{@user.login}) notifications"
+    xml.description "Event notifications from #{@configuration['title']}"
+    xml.link url_for only_path: false, controller: 'main', action: 'index'
+    xml.language "en"
+    xml.pubDate Time.now
+    xml.generator @configuration['title']
+
+    @notifications.each do |notification|
+      xml.item do
+        xml.title notification.event.subject
+        # We render the event_mailer view to 'description'
+        xml.description ApplicationController.renderer.new.render(
```

Comment:
Jupp otherwise it has no context (like format etc.)

```
ActionView::Template::Error:
       Missing template event_mailer/request_create with {:locale=>[:en], :formats=>[:rss], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :coffee, :haml]}. Searched in:
         * "/vagrant/src/api/app/views"
         * "/usr/lib64/ruby/gems/2.4.0/gems/peek-mysql2-1.2.0/app/views"
         * "/usr/lib64/ruby/gems/2.4.0/gems/peek-dalli-1.1.3/app/views"
         * "/usr/lib64/ruby/gems/2.4.0/gems/peek-1.0.1/app/views"
         * "/usr/lib64/ruby/gems/2.4.0/gems/kaminari-core-1.0.1/app/views"

```